libs/net/dal/Services/ContentService.cs

```diff
@@ -265,6 +266,25 @@ public async Task<IPaged<API.Areas.Services.Models.Content.ContentModel>> FindWi
                 .Query(filter.Keyword.ToLower())
             ));
 
+        if (!string.IsNullOrWhiteSpace(filter.Names))
+        {
+            var names = filter.Names.Split(',').Select(a => a.Trim()).ToList();
+            List<string> nameQueries = new();
+            foreach (var name in names)
+            {
+                var nameSegments = name.Split(' ');
+                var firstName = nameSegments.First();
+                var firstInitial = firstName.Substring(0, 1);
+                var lastName = nameSegments.Last();
+                //(eby +(david | d.))
+                nameQueries.Add($"({lastName} +({firstName} | {firstInitial}.))");
+            }
+            filterQueries.Add(s =>
+                s.SimpleQueryString(qs => qs
+                .Fields(f => f.Field(p => p.Headline, 10).Field(p => p.Byline, 10).Field(p => p.Body).Field(p => p.Summary))
+                .Query(string.Join(" | ", nameQueries).ToLowerInvariant())));
```

Comment:
What's the point of using `ToLowerInvariant()`?

Reply:
An old habit I had from a previous life where we were consuming text in multiple languages.  Everything had to be invariant culture.  It's not relevant here.

Reply:
removed the `ToLower()` altogether and it seems to work fine